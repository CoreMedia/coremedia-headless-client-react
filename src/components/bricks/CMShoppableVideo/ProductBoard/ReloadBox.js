// @flow
// import PropTypes from 'prop-types';
import styled from 'styled-components';

import { media } from '../../../../styles/themes/utils';

const ReloadBox = styled.div`
  display: none;
  ${props => media(props.theme.breakpoints.tablet)`
    display: block;
    position: absolute;
    bottom: 3px;
    left: 0;
    z-index: 9999;
  `};
`;

ReloadBox.displayName = 'ReloadBox';

/*ReloadBox.propTypes = {
  theme: PropTypes.shape({
    breakpoints: PropTypes.shape({
      tablet: PropTypes.number.isRequired,
    }).isRequired,
  }),
};*/

ReloadBox.defaultProps = {
  theme: {
    breakpoints: {
      tablet: 768,
    },
  },
};

export default ReloadBox;
