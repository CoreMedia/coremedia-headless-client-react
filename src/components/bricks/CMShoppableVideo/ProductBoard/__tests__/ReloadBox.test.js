// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import ReloadBox from '../ReloadBox';

describe('ReloadBox Component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<ReloadBox>Content</ReloadBox>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
