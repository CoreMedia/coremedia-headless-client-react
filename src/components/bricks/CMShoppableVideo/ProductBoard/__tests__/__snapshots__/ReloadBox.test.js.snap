// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReloadBox Component should render correctly 1`] = `
.c0 {
  display: none;
}

@media (min-width:48em) {
  .c0 {
    display: block;
    position: absolute;
    bottom: 3px;
    left: 0;
    z-index: 9999;
  }
}

<div
  className="c0"
>
  Content
</div>
`;
